public/app/features/logs/components/LogDetailsRow.tsx

```diff
@@ -93,13 +93,25 @@ class UnThemedLogDetailsRow extends PureComponent<Props, State> {
     if (onClickFilterLabel) {
       onClickFilterLabel(parsedKey, parsedValue);
     }
+
+    reportInteraction('grafana_explore_logs_log_details_filter_clicked', {
+      datasourceType: this.props.row.datasourceType,
+      filterType: 'include',
+      logRowUid: this.props.row.uid,
+    });
   };
 
   filterOutLabel = () => {
     const { onClickFilterOutLabel, parsedKey, parsedValue } = this.props;
     if (onClickFilterOutLabel) {
       onClickFilterOutLabel(parsedKey, parsedValue);
     }
+
+    reportInteraction('grafana_explore_logs_log_details_filter_clicked', {
+      datasourceType: this.props.row.datasourceType,
```

Comment:
`this.props` is desctructured a couple of lines above those. I think it's worth also to desctructure `row`.